### PR TITLE
support runner.entrypoint parameter

### PR DESCRIPTION
**Pull request by cw-ozaki**
Added parameters for overwriting docker `--entrypoint` option.

```
runner:
  image: mesosphere/aws-cli:1.14.5
  command: /bin/sh
  args: [-c]
script: echo 111;
```

```
DEBU[0000] script step started                           cmd="[docker run --rm -i --entrypoint  mesosphere/aws-cli:1.14.5 /bin/sh -c echo 111;]"
INFO[0000] 111    
```

### CODE REVIEW

File: pkg/steps/script_step.go [hunk 1]
```diff
@@ -163,9 +167,11 @@ tar zxvf %s.tgz 1>&2
 		}
 		var args []string
 		args = append(args, dockerArgs...)
+		args = append(args, "--entrypoint", c.Entrypoint)
```
Comment (mumoshu) [inline pkg/steps/script_step.go]: This seems to make it impossible to use the default entrypoint specified in the image. How about making this conditional, by adding `--entrypoint ENT` only when it isn't empty, like this?

```golang
if c. Entrypoint != "" {
    dockerArgs = append(dockerArgs, "--entrypoint", c. Entrypoint)
}
```

**Comment by cw-ozaki** [comment]
@mumoshu 
fixed it.

```
runner:
  image: mesosphere/aws-cli:1.14.5
  entrypoint: /bin/sh
  command: -c
```

When overwriting `entrypoint`, we need to write like this, is it OK?

**Comment by mumoshu** [comment]
@cw-ozaki Ah, good point! How about allowing `nil` in it, so that you can differentiate between unspecified(`entrypoint: nil` or omitted, which means to use the default entrypoint specified in the image) and emptied(`entrypoint: ""`)?


```
Entrypoint *string
```

And

```
if c.Entrypoint != nil {
    dockerArgs = append(dockerArgs, "--entrypoint", c.Entrypoint)
}
```

**Comment by cw-ozaki** [comment]
@mumoshu 
That's nice! fixed it.

**Pull request merged**